src/main/java/seedu/address/logic/commands/AddContactCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+
+/**
+ * Adds a contact to Teaching Assistant.
+ */
+public class AddContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
```

Comment:
Command word should be "cadd", have to handle this commandword in the AddressBookParser also